### PR TITLE
Bring release/1.3.0 changes to develop

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,8 +66,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>2.8.0</cdap.version>
-    <cdap.clients.version>1.1.0-SNAPSHOT</cdap.clients.version>
+    <cdap.version>3.0.0</cdap.version>
```
Comment (gsps1) [inline pom.xml]: 3.1.0

**Comment by anwar6953** [comment]
Was there a need to bump the release version? New features / improvements?

**Comment by anwar6953** [comment]
Oh, just to depend on the new version of cdap, I guess?

**Comment by awholegunch** [comment]
Yes...new version of cdap...and then the develop branch should get bumped to the SNAPSHOT.

**Comment by anwar6953** [comment]
ok. LGTM

**Pull request merged**